modules/images-other-jenkins-agent-images.adoc

```diff
@@ -15,6 +15,8 @@ $ docker pull registry.redhat.io/openshift4/ose-jenkins:<v4.2.0>
 $ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.2.0>
 $ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.2.0>
 $ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.2.0>
+$ docker pull registry.redhat.io/openshift4/jenkins-agent-nodejs-10-rhel7:<v4.3.0>
+$ docker pull registry.redhat.io/openshift4/jenkins-agent-nodejs-12-rhel7:<v4.3.0>
```

Comment:
@jboxman the code on line 18 and 19 should replace the code on line 15.
Also, I think we should change <v4.3.0> to <v4.2.0> to match the other mentions here.  
Also, in the 4.5 version and the 4.4 version of the docs, the images have <v4.3.0>. Should this be changed to the respective version for all the images?
Also, this PR needs to be cherrypicked to 4.1 where we have <v4.1.4>

@jkhelil @akram @waveywaves; noob question, do these versions add any value? If not, can we remove them? If yes, @jboxman is using the Version attribute possible here? 
CC: @bmcelvee Since I see you mentioned in earlier versions, can you let us know what you think, please?

Reply:
For specific versions, those specific version based images would be pulled, I think this is important to furture proof ourselves if there are changes which aren't backported to previous versions of the Openshift.  I agree that the 4.3 needs to be changed to 4.2 here to match the versions.

Reply:
@Preeticp @waveywaves 

Updated.